lib/ansible/module_utils/network/nxos/config/l2_interfaces/l2_interfaces.py

```diff
@@ -274,6 +278,8 @@ def set_commands(self, w, have, replace=False):
         else:
             diff = self.diff_of_dicts(w, obj_in_have)
             if diff and not replace:
+                if 'mode' in diff.keys() and diff['mode']:
+                    commands = self.add_commands(diff, True)
```

Comment:
'True' need not be passed as an argument for 'add_commands'

Reply:
+1